elements/rh-accordion/rh-accordion-header.css

```diff
@@ -0,0 +1,114 @@
+:host {
+  --_padding-block-start: var(--rh-space-lg, 16px);
+  --_padding-inline-end: var(--rh-space-xl, 24px);
+  --_padding-block-end: var(--rh-space-lg, 16px);
+  --_padding-inline-start: var(--rh-space-xl, 24px);
```

Comment:
would be good to have a RTL demo. there's an example in CTA

Reply:
I added an [RTL demo](https://deploy-preview-594--red-hat-design-system.netlify.app/components/accordion/demo/right-to-left/). it looks ok-but-not-great. We can merge this and loop back, provided this comment gets spun out to a new issue

TODO:
- adjust rotate direction
- align texts (there's maybe some flexbox thing going on)
- change border alignment